src/services/identity/DeploymentsService.ts

```diff
@@ -43,32 +43,55 @@ class DeploymentsService {
     repoName: string
     site: Site
   }): Promise<Deployment> => {
-    const amplifyResult = await this.createAmplifyAppOnAws(repoName)
-    if (amplifyResult.isErr()) {
-      logger.error(`Amplify set up error: ${amplifyResult.error}`)
-      throw amplifyResult.error
+    const [
+      amplifyStagingResult,
+      amplifyStagingLiteResult,
+    ] = await this.createAmplifyAppsOnAws(repoName)
+    if (amplifyStagingResult.isErr()) {
+      logger.error(`Amplify set up error: ${amplifyStagingResult.error}`)
+      throw amplifyStagingResult.error
     }
-    const amplifyInfo = amplifyResult.value
+
+    if (amplifyStagingLiteResult.isErr()) {
+      logger.error(`Amplify set up error: ${amplifyStagingLiteResult.error}`)
```

Comment:
Should we differentiate between this error message and the previous one? So that it is easier at a glance to know which app creation failed.

Reply:
fair point 9fbbd471bd21ea9251aaffb7e7f4eabfa8794afe